### PR TITLE
Add terraform for creating master nodes

### PR DESCRIPTION
**Pull request by mjturek**
Adds terraform for creating master nodes and outputting their IPs.

### CODE REVIEW

File: data/data/powervs/master/main.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  pi_network_name      = var.network_name
+  pi_cloud_instance_id = var.cloud_instance_id
+}
+
+data "ibm_pi_image" "master_image" {
+  pi_image_name        = var.image_name
+  pi_cloud_instance_id = var.cloud_instance_id
+}
+
+# Create the bootstrap instance
```
Comment (clnperez) [inline data/data/powervs/master/main.tf]: copy/paste fix: Create the master instance

**Review by clnperez — CHANGES_REQUESTED** [review]
LGTM aside from a small nit. I requested @mkumatag as a reviewer as well.

File: data/data/powervs/master/main.tf (hunk 1 above)
Comment (mjturek) [reply data/data/powervs/master/main.tf]: Done.

**Comment by mjturek** [comment]
Thanks for the review! Makes sense, would like to hear from @mkumatag as well

**Comment by mjturek** [comment]
Fixed a merge conflict.

File: data/data/powervs/master/main.tf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  pi_processors        = var.processors
+  pi_instance_name     = "${var.cluster_id}-master-${count.index}"
+  pi_proc_type         = var.proc_type
+  pi_image_id          = data.ibm_pi_image.master_image.id
+  pi_sys_type          = var.sys_type
+  pi_cloud_instance_id = var.cloud_instance_id
+  pi_network_ids       = [data.ibm_pi_network.network.id]
+
+  pi_user_data         = base64encode(var.ignition)
+  pi_key_pair_name     = "${var.cluster_id}-keypair"
```
Comment (mkumatag) [inline data/data/powervs/master/main.tf]: Not sure if we are creating this dummy key somewhere! if not, lets create this somewhere just to satisfy the PowerVS APIs

**Review by mkumatag — CHANGES_REQUESTED** [review]
one minor comment, otherwise lgtm

File: data/data/powervs/master/main.tf (hunk 2 above)
Comment (mjturek) [reply data/data/powervs/master/main.tf]: ahhh we are not, I made it manually. I can throw in a key creation block!

**Comment by clnperez** [comment]
LGTM

**Pull request merged**